doc/gettingStarted/Watchfaces.md

```diff
@@ -0,0 +1,29 @@
+# Watchfaces
+
+InfiniTime has 6 apps on the `main` branch at the time of writing.
+
+## List of apps
+- Digital
+- Analog
+- PineTimeStyle
+- Terminal
+- Infinineat
+- Casio G7710
+
+### Digital
+![Digital face](/doc/gettingStarted/Watchfaces/Digital.png)
+
+### Analog
+![Analog face](/doc/gettingStarted/Watchfaces/Analog.png)
+
+### PineTimeStyle
+![PineTimeStyle face](/doc/gettingStarted/Watchfaces/PineTimeStyle.png)
```

Comment:
It's a good opportunity to mention that user can long-tap the display to configure the watchface  (colors, step style, weather)